### PR TITLE
Pruning

### PR DESCRIPTION
**Pull request by vladdez**
What is currently done
- added new examples into [butterfly.md](https://github.com/unfoldtoolbox/UnfoldMakie.jl/pull/142/commits/5cdd1df229708c3ba217a89eac74a830700e6d13#diff-f21550bf16468857d39602df2ca17411f52cfa9b8dcf53a3996753de119fd3e9)
- the bug of "underscores converted into italic" in docstrings was solved using isapporx()  
- new tests for erp plot, erp image. Little cleaning of some other tests
- formatting and text correction
- solving an [assert error](https://github.com/JuliaDocs/Documenter.jl/pull/2388)
- circTopo renamed to circ_topo
- resolved problem with pvlaues in docs about erp
- resolved problems with empty examples in docs
- push_preview enabled (now we can see docs in pull request)

### CODE REVIEW

File: src/plot_circulareegtopoplot.jl [hunk 1]
```diff
@@ -113,7 +113,7 @@ function calculateGlobalMaxValues(data, predictor)
     )
 
 
-    globalMaxVal = maximum(x.localMaxVal)
+    globalMaxVal = round(maximum(x.localMaxVal), digits = 2)
```
Comment (behinger) [inline src/plot_circulareegtopoplot.jl:116]: I dont think this is a good idea. why? what if localMaxVal is a very small number?

Comment (vladdez) [reply src/plot_circulareegtopoplot.jl:116]: You mentioned some approximation function
Is it ApproxFun from https://github.com/unfoldtoolbox/UnfoldSim.jl/pull/70

**Comment by vladdez** [comment]
What is currently done
- added new examples into [butterfly.md](https://github.com/unfoldtoolbox/UnfoldMakie.jl/pull/142/commits/5cdd1df229708c3ba217a89eac74a830700e6d13#diff-f21550bf16468857d39602df2ca17411f52cfa9b8dcf53a3996753de119fd3e9)
- the bug with underscores converted into italic in docstrings was solved
- new tests for erp plot, erp image. Little cleaning of some other tests
- formatting
- solving an [assert error](https://github.com/JuliaDocs/Documenter.jl/pull/2388)

File: src/plot_circulareegtopoplot.jl (hunk 1 above)
Comment (behinger) [reply src/plot_circulareegtopoplot.jl:116]: the function is literally called `approx` - maybe googling "Floating point comparison approx julia` or something can give you some context. 

I think you need approx also not here, but in the unit test, correct?

File: src/docstringtemplate.jl [hunk 2]
```diff
@@ -5,46 +5,48 @@ function _docstring(cfg_symb::Symbol)
...
-        out = out * "**$(fn[k]) =** $(string(namedtpl)) $addlink \n\n"
+
+        out = out * "**$(fn[k]) =** $(string(`namedtpl`)) $addlink \n\n"
     end
-        
+
     return """## Shared plot configuration options
         The shared plot options can be used as follows:
-        `type = (; key=value, ...))` - for example `plot_x(..., layout = (;show_legend=true, legend_position=:right))`. 
+        `type = (; key=value, ...))` - for example `plot_x(...; layout = (show_legend=true, legend_position=:right))`.
```
Comment (behinger) [inline src/docstringtemplate.jl:42]: can you add the `(;show_legend=true)` again?

File: src/docstringtemplate.jl [hunk 3]
```diff
@@ -5,46 +5,48 @@ function _docstring(cfg_symb::Symbol)
...
         :colorbar => "use `kwargs...` of  $cbarstring",
         :visual => "use `kwargs...` of [`$(visuallink[cfg_symb])`](@ref)",
-        
     )
     for k = 1:length(fn)
         namedtpl = Base.getfield(cfg, fn[k])
         addlink = ""
         try
-            addlink = "- *"*link[fn[k]]*"*"
+            addlink = "- *" * link[fn[k]] * "*"
```
Comment (behinger) [inline src/docstringtemplate.jl:33]: this supposedly adresses the italic, but I think the wrong italic. the issue is this italic:
grafik

not this 
grafik


I'd rather keep the latter actually.

File: src/docstringtemplate.jl (hunk 2 above)
Comment (behinger) [reply src/docstringtemplate.jl:42]: (the `;`)

File: src/plot_circulareegtopoplot.jl [hunk 4]
```diff
@@ -113,6 +113,7 @@ function calculateGlobalMaxValues(data, predictor)
     )
 
 
+    #globalMaxVal = round(maximum(x.localMaxVal), digits = 2)
```
Comment (behinger) [inline src/plot_circulareegtopoplot.jl:116]: can you remove the line?

**Review by behinger — COMMENTED** [review]
Some very sensible changes. Thanks!

File: src/docstringtemplate.jl (hunk 3 above)
Comment (vladdez) [reply src/docstringtemplate.jl:33]: image

the solution is on the line 37
on 32 is just an action of JuliaFormatter

Comment (behinger) [reply src/docstringtemplate.jl:33]: no. formater broke it that is why you dont see the italics ;)

Comment (behinger) [reply src/docstringtemplate.jl:33]: that is also why it now says axis = namedtpl instead of the actual content

Comment (vladdez) [reply src/docstringtemplate.jl:33]: ah I see, indeed
any ideas how to solve it?

Comment (behinger) [reply src/docstringtemplate.jl:33]: definitely revert the formatter change - I have to think about it

**Comment by behinger** [comment]
Good to go from my side :) the documenter preview link also worked for me!

**Comment by behinger** [comment]
maybe in a dedicated new PRs to keep them small?

**Comment by vladdez** [comment]
Yeah let's do it although I already did it for butterfy plot to test

**Pull request merged**